modules/host/persist.go

```diff
@@ -25,34 +26,42 @@ var persistMetadata = persist.Metadata{
 
 // persistence is the data that is kept when the host is restarted.
 type persistence struct {
-	// Host Context.
+	// Consensus Tracking.
 	BlockHeight  types.BlockHeight
 	RecentChange modules.ConsensusChangeID
-	PublicKey    types.SiaPublicKey
-	SecretKey    crypto.SecretKey
-	Settings     modules.HostSettings
+
+	// Host Identity.
+	NetAddress modules.NetAddress
+	PublicKey  types.SiaPublicKey
+	SecretKey  crypto.SecretKey
 
 	// File Management.
-	FileCounter int64
-	Obligations []contractObligation
+	Obligations []*contractObligation
 
 	// Statistics.
-	Profit types.Currency
+	FileCounter int64
+	LostRevenue types.Currency
+	Revenue     types.Currency
+
+	// RPC Tracking.
+	ErroredCalls   uint64
+	MalformedCalls uint64
+	DownloadCalls  uint64
+	RenewCalls     uint64
+	ReviseCalls    uint64
+	SettingsCalls  uint64
+	UploadCalls    uint64
```

Comment:
grouping these into a "metrics" struct is probably a good idea -- could be used in the Host type as well.